docs/react-testing-library/api.mdx

```diff
@@ -107,6 +107,8 @@ your components.
 
 ### `legacyRoot`
 
+**WARNING**: This option is only available when tests run with React 18 and earlier.
```

Comment:
AFAIR and please correct me if I'm wrong, using `legacyRoot` wasn't available for versions before React 18. Also, I think a note here will be nicer in terms of UX.
```suggestion
:::note
This option is only available when tests run with React 18.
:::
```

Reply:
Legacy root is everything before React 18. We don't officially support React 17 with latest RTL though. But this section is specifically for the React version you have installed not for the RTL version.

Reply:
Also: this is not a note but a warning. Feel free to follow-up with an appropriate styling that's specific to our deploy setup.